packages/muelu/test/unit_tests/CoalesceDropFactory.cpp

```diff
@@ -1223,7 +1223,7 @@ TEUCHOS_UNIT_TEST_TEMPLATE_4_DECL(CoalesceDropFactory, DistanceLaplacianScaledCu
   // L_ij = -36
   // L_ii = 72
   // criterion for dropping is |L_ij|^2 <= tol^2 * |L_ii*L_jj|
-  coalesceDropFact.SetParameter("aggregation: drop tol", Teuchos::ParameterEntry(8.0));
+  coalesceDropFact.SetParameter("aggregation: drop tol", Teuchos::ParameterEntry(0.125));
```

Comment:
@NyquilDreams 
Please correct me if I am wrong, but I don't think that this is not the correct threshold.
You replaced a criterion
```
a > realThreshold * b
```
with
```
realThreshold * realThreshold * a > b
```
Doesn't that mean that the threshold should tranform like 
```
threshold -> 1/sqrt(threshold)
```
not
```
threshold -> 1/threshold
```
in order to preserve behavior?

Reply:
You are correct. I will make a change accordingly.